pkg/catalog/install.go

```diff
@@ -88,12 +121,53 @@ func Install(cfg InstallConfig, gitClient git.Git) error {
 	return generateOutput(filepath.Join(profileRootdir, "profile.yaml"), &subscription)
 }
 
+// cloneGitRepositoryObjects takes a GitRepository objects and clones what it points to, then discards the object.
+func cloneGitRepositoryObjects(cfg InstallConfig, g *sourcev1.GitRepository, artifactDir string) error {
```

Comment:
does this mean we do a `git clone` per artifact that we need to copy? also the naming is confusing, wb `getGitRepositoryArtifact` or something, the fact that it clones doesn't matter :smile: